src/lib/elements/table/tableScroll.svelte

```diff
@@ -2,6 +2,7 @@
     import type { Action } from 'svelte/action';
 
     export let isSticky = false;
+    export let noMargin = false;
```

Comment:
any reason we use `noMargin=false` instead of `margin=true` ? 
It's harder to parse through conditions when using double negatives.

Reply:
While I do agree, this is something that's present console wise. If we change it here, we would have to change it everywhere, which is fine by me, but I'd like to run it by Torsten.

Also, if we do change it, I'd argue for hasMargin instead of just margin. Margin sounds like I'm going to pass in a numeric value

Reply:
Yea sounds fair 👍  if this pattern is already being followed we can delay this decision for later.